src/adapter/hook.ts

```diff
@@ -117,7 +120,7 @@ export function createHook(bridge: Bridge): DevtoolsHook {
 				listeners[idx] = null;
 			};
 		},
-		attachPreact: (version, options, config) => {
+		attachPreact: (version, options, config, minDevtoolsVersion) => {
```

Comment:
Thinking a bit more about this, I'm unsure if we should go down the route of forcing Preact to know about the specific devtools version. I'm worried that a change may slip into core, we'll do a release, but we forgot to change the minimum devtools version there.

The approach with a minimum devtools version is solid imo :+1: What are you're thoughts on this? Maybe we need to play ping pong with the idea a few times until we land on an elegant solution :)